static/app/views/replays/replayTable/index.tsx

```diff
@@ -69,6 +72,40 @@ function ReplayTable({
     .filter(Boolean)
     .map(column => <HeaderCell key={column} column={column} sort={sort} />);
 
+  const {
+    selection: {projects},
+  } = usePageFilters();
+
+  const MIN_DEAD_RAGE_CLICK_SDK = '7.60.1';
+
+  const needSDKUpgrade = useProjectSdkNeedsUpdate({
+    minVersion: MIN_DEAD_RAGE_CLICK_SDK,
+    organization,
+    projectId: projects.map(String),
+  });
+
+  if (needSDKUpgrade.needsUpdate) {
+    return (
+      <StyledPanelTable
+        headers={tableHeaders}
+        isLoading={false}
+        visibleColumns={visibleColumns}
+        data-test-id="replay-table"
+        gridRows={undefined}
+      >
+        <StyledAlert type="info" showIcon>
+          {tct('[data] require an [sdkPrompt]. [link:Learn how to upgrade.]', {
```

Comment:
I'm not sure if the sentence makes sense as-is. I see that we have `require an` inside replayList.tsx too. Maybe i'm sensitive, idk

This is what I would try:

```suggestion
          {tct('[data] requires [sdkPrompt]. [link:Learn how to upgrade.]', {
```

Reply:
we can say 'the rage and dead click table requires' ?

Reply:
`requires` reads better to me.

Reply:
so keep as is @billyvg ?

Reply:
@michellewzhang "Rage and dead clicks requires SDK version >= 7.60.1"